docs/UserGuide.md

```diff
@@ -144,52 +158,58 @@ Examples:
 
 Updates a patient's details from the GoMedic application.
 
-Format: 
-`update t/patient i/PATIENT_ID [OPTIONAL_PARAMETER]...`
+Format: `update t/patient i/PATIENT_ID [OPTIONAL_PARAMETER]...`
+
+The parameters are:
 
-The flags are:
 * `n/NAME` indicates the full name of the patient, first name and last name are separated by `-`.
 * `a/AGE` is greater than or equal to 0.
 * `g/GENDER` is chosen from one of 3 choices, `M/F/O` where `M` is for Male, `F` is for Female, and `O` is for Others.
 * `h/HEIGHT` is the height of patient in centimeters rounded to the nearest integer.
 * `w/WEIGHT` is the weight of patient in kilograms rounded to the nearest integer.
 * `b/BLOOD_TYPE` is chosen from one of the 4 choices, `A/B/AB/O`.
 * `c/CONTACT_NUMBER` must be 8-digit Singapore phone number.
-* `o/[OPTIONAL...]` is the list of patient's past/pre-existing medical conditions. For medical condition that has
+
+The optional parameters are: 
+* `o/description` is the list of patient's past/pre-existing medical conditions to be **added**. For medical condition that has
   multiple words, use `-` to combine the words, e.g. `heart-failure`. To separate between conditions, use `/`.
-* `do/[OPTIONAL_TO_DELETE...]` is the list of patient's past/pre-existing medical conditions to delete. For medical
-  condition that has multiple words, use `-` to combine the words, e.g. `heart-failure`. To separate between conditions, use `/`.
+* `do/description` is the list of patient's past/pre-existing medical conditions to be **deleted**. For medical
+  condition that has multiple words, use `-` to combine the words, e.g. `heart-failure`. 
+
+Notes: 
 * Patient ID can be obtained by listing all the patients or search for a certain patients with available filters.
 * Patient ID is **unique** (i.e. every patient will be assigned to a unique ID, hence this guarantees
   1 `delete t/patient` command will not delete 2 patients at once).
 * Invalid Patient ID being supplied would be flagged by GoMedic, and do not cause changes to any existing patients.
-* Invalid `OPTIONAL_TO_DELETE` conditions supplied would be flagged by GoMedic, and do not cause changes to the
-existing patient.
+* Invalid `OPTIONAL_TO_DELETE` conditions supplied would be flagged by GoMedic, and do not cause changes to the existing
+  patient.
 
 Examples:
 
 * `update t/patient i/P123 n/John-Doe a/30 g/M`
 * `update t/patient i/P003 n/Tom-Doe a/20 g/M h/167 w/61 b/AB c/12341234 do/diabetes`
 
```

Comment:
Can add example that deletes 2 medical conditions and add medical condition as well to give a better idea to the user.
e.g. update t/patient i/P003 n/Tom-Doe a/20 g/M h/167 w/61 b/AB c/12341234 do/diabetes do/heart-feailure

Reply:
yup sure, I add also combination with o/ flag